types/draft.ts

```diff
@@ -12,19 +12,19 @@ const Draft = z.object({
 type Draft = z.infer<typeof Draft>
 
 const PendingDraft = z.object({
-	title: z.string().min(1).max(100),
-	summary: z.string().min(10).max(255),
-	description: z.string().min(1).max(1000),
+	title: z.string().max(48),
+	summary: z.string().max(255),
+	description: z.string().max(2048),
 	type: z.enum(['topic', 'project']),
 })
 
 type PendingDraft = z.infer<typeof PendingDraft>
 
-const DraftUpdate = z.object({
-	title: z.string().min(1).max(100).optional(),
-	summary: z.string().min(10).max(255).optional(),
-	description: z.string().min(1).max(1000).optional(),
-	type: z.enum(['topic', 'project']).optional(),
+const DraftUpdate = PendingDraft.partial({
```

Comment:
You might need to test this out but I'm pretty sure since we're making ALL the properties optional you can just call `PendingDraft.partial()` and not need to pass `true` for all the fields. I think the fields object gives you the option to make only some fields optional, but in this case we want them all so the default is probably cleaner in the long run.